packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss

```diff
@@ -265,4 +265,73 @@ cxl-vaadin-accordion {
       }
     }
   }
+
+  /**
+   * Theme "cxl-dashboard-category"
+   */
+   &[theme~="cxl-dashboard-category"] {
+    .header {
+      display: flex;
+      align-items: baseline;
+      padding: var(--lumo-space-l) var(--lumo-space-m);
+
+      @media #{mq.$small} {
+        padding: var(--lumo-space-l);
+      }
+
+      .entry-title {
+        margin: 0;
+      }
+
+      > span {
+        margin: 0 0 2px var(--lumo-space-m);
```

Comment:
How big impact `2px` margin has here? I tested in Chrome with `margin-bottom: 0px;` and I don't see any difference.